Source/WebKit/UIProcess/API/Cocoa/WKWebViewConfigurationInternal.h

```diff
@@ -50,3 +53,13 @@ _WKDragLiftDelay toDragLiftDelay(NSUInteger);
 _WKDragLiftDelay toWKDragLiftDelay(WebKit::DragLiftDelay);
 WebKit::DragLiftDelay fromWKDragLiftDelay(_WKDragLiftDelay);
 #endif
+
+#endif // __cplusplus
+
+@interface WKWebViewConfiguration (NonCpp)
```

Comment:
```suggestion
@interface WKWebViewConfiguration ()
```
You should declare this as a class extension (not a named category). Categories should technically have their implementation supplied in a matching `@implementation Foo (Bar)` block, where you're just implementing it in the primary class implementation, but also there's just no reason for it to be category when you're the defining framework for a type.